packages/ui-components/src/components/inline-editable-field/inline-editable-field.tsx

```diff
@@ -81,6 +81,15 @@ export class KvInlineEditableField {
 		}
 	}
 
+	@Watch('disabled')
+	handleDisableChange(state) {
```

Comment:
This is not called when this component is "mounted", not sure if we need to add it to `connectedCallback`

Reply:
Okok 👍